app/views/tasks/_form.html.erb

```diff
@@ -43,4 +43,12 @@
   <div class="actions">
     <%= f.submit(nil, class: "btn btn-primary") %>
   </div>
+  <div>
+    <% if @task.id == nil %>
+      <%= link_to 'Back', dashboard_path, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger' %>
```

Comment:
We need to check that I'm sure that I want to go back? I can understand the sentiment of "This stuff won't be saved if you leave," so I would suggest changing this to something that sounds more like that.